recipes/cymunk/__init__.py

```diff
@@ -8,46 +8,20 @@
 from os.path import join
 import sh
 
+
 class CymunkRecipe(CythonRecipe):
     version = 'master'
-    url = 'https://github.com/tito/cymunk/archive/{version}.zip'
+    url = 'https://github.com/kivy/cymunk/archive/{version}.zip'
     name = 'cymunk'
-    depends = ['hostpython']
-    cythonize = True
+    pre_build_ext = True
+    library = 'libcymunk.a'
+
+    depends = ['python3', 'hostpython']
```

Comment:
Is this saying it does *not* work on python2? or could this be `depends = ['python', 'hostpython']`? (or even, `depends = ['python']`)

Reply:
fixed. Also tested if it builds with python2 - it builds. (however, I didn't test if it work with py2).